src/models/event.rs

```diff
@@ -16,6 +16,15 @@ pub struct Event {
 }
 
 impl Event {
+    pub fn new() -> Event {
```

Comment:
Doesn't really make sense to me to have a `new()` function which only sets default values and doesn't take any args, better then to implement the Default trait for Event and then do `Event { ..Default::default() }`

Reply:
Huh, had no idea about that. Might give it a shot.